Dell/recovery_common.py

```diff
@@ -655,12 +691,11 @@ def parse(fname):
                     arch, package, modaliases = parse(os.path.join(repo, fname))
                     if not modaliases and (arch == "all" or arch == my_arch):
                         to_install.append(package)
-
```

Comment:
for this and others, don't change whitespace in the commit.  If whitespace was wrong previously, please have a separate follow up to fix it from your actual code changes commits.

Reply:
Got it, thanks.